Project.toml

```diff
@@ -17,7 +17,7 @@ Extents = "0.1"
 GeoFormatTypes = "0.4"
 GeoInterface = "1"
 GeoInterfaceRecipes = "1"
-JSON3 = "1"
+JSON3 = "1.12"
```

Comment:
Should this be 1.1? Note that real breaks need v2. But, I'm not sure if this is really breaking, it seems more like fixing a bug to me.

(And  I generally find it better not to bump versions in PRs)

Reply:
This is the JSON3 version, not GeoJSON. I didn't bump the GeoJSON version yet.

Reply:
About breaking/not breaking, I agree for coordinates it's definitely bugfix. For integer properties it could be a minor change, I'm also fine with releasing as non-breaking.

Reply:
Lol I never read toml files correctly in PRs. But yes, bugfix it is.